dev-tools/reconfigurator-cli/tests/output/cmds-set-mgs-updates-stdout

```diff
@@ -1,6 +1,5 @@
 using provided RNG seed: reconfigurator-cli-test
 > # Load example system
-
 > # This is a legacy test; new tests shouldn't need to set a seed
```

Comment:
This is where I think it's an improvement -- I find it much easier to read without the extra line in between.